README.md

```diff
@@ -2,23 +2,83 @@
 
 # mobx-state-tree
 
-_Opinionated, transactional, MobX powered state container combining the best features of the immutable and mutable world for an optimal DX_
-
 [![npm version](https://badge.fury.io/js/mobx-state-tree.svg)](https://badge.fury.io/js/mobx-state-tree)
 [![CircleCI](https://circleci.com/gh/mobxjs/mobx-state-tree.svg?style=svg)](https://circleci.com/gh/mobxjs/mobx-state-tree)
 [![Coverage Status](https://coveralls.io/repos/github/mobxjs/mobx-state-tree/badge.svg?branch=master)](https://coveralls.io/github/mobxjs/mobx-state-tree?branch=master)
-[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/mobx-state-tree)
+[![Join the community on Github Discussions](https://img.shields.io/badge/GitHub-Discussions-lightgrey)](https://github.com/mobxjs/mobx-state-tree/discussions)
+
+## What is mobx-state-tree?
+
+Technically speaking, mobx-state-tree (also known as MST) is a state container system built on [MobX](https://github.com/mobxjs/mobx), a functional reactive state library.
+
+This may not mean much to you, and that’s okay. I’ll explain it like this: **MobX is a state management "engine", and MST makes it feel a lot more like Redux by giving it structure.** MST also offers better performance and much less boilerplate than Redux!
```

Comment:
As a person, having extensive experience in both I would say opposite: MST feels a lot not like Redux, which one of the things we choose it over Redux, it has different architecture, philosophy and mind models, agreed on the following sentence. Also none of your following examples show the similarity you mention.

I would rephrase it in a way:

`MST is MobX on steroids, MST gives structure, patterns and recipes, which are valuable in large team, but also useful is small one when you expect your code to scale rapidly. If we compare it to Redux, MST offers better performance and much less boilerplate code than Redux!`

Thank you for taking care of it.

Reply:
@Romanior I like your changes! I'll make some modifications accordingly.